setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='OpenFisca-Aotearoa',
-    version='9.1.0',
+    version='9.3.0',
```

Comment:
> 9.2.0 is severely disappointed at being overlooked

Reply:
There was a 9.2.0 release, but the setup.py file wasn't updated